### DISCUSSION TITLE
Name of color theme used on marketing website for copilot

### DISCUSSION THREAD

**Discussion by ujvaljoshi** [community/community#106599] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Possible to know name of the theme which is used in copilot website. Tried to search but couldnt find name.

![Screenshot 2024-02-07 at 10 47 36@2x](https://github.com/community/community/assets/870044/0b24fbd6-afb0-433d-9e70-0cc892ff4f9e)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ujvaljoshi ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Comment by Piyusss** [comment]
There is no specific name of it. Simply search 'Copilot Theme' or, You may can refer to this -
https://marketplace.visualstudio.com/items?itemName=BenjaminBenais.copilot-theme
